packages/farms/constants/324.ts

```diff
@@ -52,4 +52,25 @@ export const farmsV3 = defineFarmV3Configs([
     token1: zksyncTokens.usdc,
     feeAmount: FeeAmount.LOWEST,
   },
+  {
+    pid: 8,
+    lpAddress: '0x662cD659F91528FF27D7Cb6Ac25e6EBA11c4003C',
```

Comment:
Wrong lpAddress 
pid 8, should be `0x7c0e7d6066af191977a4483b445b5c06fc41ecd6`

and make the data above pid1 , should be

[{pid: 11}, pid: 10}, {pid: 9}, {pid: 8}, {pid: 1}, ]